core/chain_indexer.go

```diff
@@ -108,6 +110,13 @@ func NewChainIndexer(chainDb ethdb.Database, indexDb ethdb.Database, backend Cha
 		confirmsReq: confirm,
 		throttling:  throttling,
 		log:         log.Log,
+		GetReceipts: func(blockHash common.Hash) types.Receipts { // Temp function, replaced with core.GetReceipts()
```

Comment:
Is there a reason to add this as an anonymous member function instead of an associated method?

i.e. why not the following?
```
fn (i *ChainIndexer) GetReceipts(...) {...}
```

Reply:
Fixed